lib/dealership.rb

```diff
@@ -0,0 +1,42 @@
+class Dealership
+
+  attr_reader :inventory
+  def initialize(name, address)
+  @name = name
+  @address = address
+  @inventory = []
+  @total_value = 0
+  end
+
+  def inventory_count
+    @inventory.count
+  end
+
+  def add_car(car)
+    @inventory << car
+  end
+
+  def has_inventory?
+    @inventory.count > 1
+  end
+
+  def cars_by_make(maker)
+    @inventory.each do |car|
+    if  car.make == maker
+      return car
+    end
+    end
+  end
```

Comment:
You want to be returning an array here rather than 1 car object

```suggestion
  def cars_by_make(maker)
    results = []
    @inventory.each do |car|
      if car.make == maker
        results << car
      end
    end
    results
  end
```